types/global.d.ts

```
@@ -98,6 +98,44 @@ declare global {
+  type FinnhubSearchResponse = {
+    count: number;
+    result: FinnhubSearchResult[];
+  };
+
+  type SearchCommandProps = {
+    renderAs?: "button" | "text";
+    label?: string;
+    initialStocks: StockWithWatchlistStatus[];
+  };
+
+  type SearchCommandProps = {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'SearchCommandProps'. Consider to delete it or rename it.

'SearchCommandProps' is defined here:


(lint/suspicious/noRedeclare)